WordPress/twentytwentyfour
# Adjust single with sidebar template

**Description**

I've updated the Single with Sidebar template to get the sidebar close to 360 pixels as in the design. It's not exact, but removing the empty columns and adding some extra padding brings it closer to the design. The sidebar is about 340 pixels at max.

**Screenshots**


<img width="1298" alt="CleanShot 2023-10-10 at 19 23 28@2x" src="https://github.com/WordPress/twentytwentyfour/assets/7585600/eceb1f32-da2c-4e1e-b746-8b52bc861770">


**Testing Instructions**





1. Checkout this PR
2. Go to the Site Editor -> Templates
3. Make sure there are no customizations saved for the "Single with Sidebar" template
4. Open template to check in the editor
5. Open a post
6. Set the template to "Single with Sidebar"
7. Save post
8. Open saved post in the frontend
